msl-commands.c

```diff
@@ -1,6 +1,9 @@
 #include <stdio.h>
 #include "ge.h"
-#include "msl-commands.h"
+
+#ifndef MSL_COMMANDS_INCLUDED_BY_MSL_STATES
```

Comment:
does this work use the improved makefile of PR#11?
In case it doesn't,   what do we think about `static inline` functions in .h? 
 @danielinux?

Reply:
`static inline` may not work:
- too much to inline
- functions referenced as by function pointers